FilesAndExceptions_task8/src/main/java/org/Marta/PeselValidator.java

```diff
@@ -0,0 +1,18 @@
+package org.Marta;
+
+public class PeselValidator {
+    public static void validatePesel(Object peselNumber) throws IllegalLengthException, WrongTypeOfDataException {
+        if(peselNumber instanceof String)
```

Comment:
Jeżeli chodzi o to sprawdzenie typu danych to chodziło raczej o potwierdzenie, że pesel zawiera same liczby (mimo bycia stringiem) ;)

Reply:
Spoko, w takim razie sprawdzamy format, a nie typ danych ;)